app/assets/javascripts/rails_db/application.js

```diff
@@ -33,6 +33,34 @@ $(function(){
     save_expand_collapse();
     return false;
   });
+
+  $('body').on('click', '.settings-link', function() {
+    $('.rails-db-settings').toggle();
+  });
+
+  $('body').on('click', '.column-selection input, .column-name', function() {
+    table_name = $('h2')[0].innerHTML;
+    if($(this).hasClass('column-name')){
```

Comment:
remove this code. I want this JS only for JS (you need to change span to label first and put checkbox inside label)